client/Dockerfile

```diff
@@ -7,12 +7,23 @@
 # See https://github.com/nodejs/docker-node/blob/main/README.md#how-to-use-this-image
 FROM node:lts
 
-# ENV NODE_ENV=production
+# The base node image sets a very verbose log level.
+#ENV NPM_CONFIG_LOGLEVEL warn
 
-COPY ["package.json", "package-lock.json*", "./"]
+COPY . .
 
+# Build for production.
 RUN npm install --legacy-peer-deps
+RUN npm run build --production
+
+# Install `serve` to run the application.
+RUN npm install -g serve
 
 COPY . .
 
-CMD [ "npm", "start" ]
\ No newline at end of file
+# Set the command to start the node server.
+CMD serve -p 80 -s build
+
+# Tell Docker about the port we'll run on.
+# virker ikke
+EXPOSE 80
```

Comment:
Vil foreslå at det flyttes til docker-compose.yml filen måske? :slightly_smiling_face: 

Her var hvad jeg havde:
```
version: '3.8'

services:
 client:
  build:
   context: ./client
  ports:
   - 8080:3000
  expose:
   - 3000
   - 3002
  command: npm start
 server-nest:
  build:
   context: ./server-nest
  expose:
   - 3000
   - 3002
  command: npm start
  ```

`ports` er for eksponering til hosten og dermed verdenen. Her mapper vi så `8080` fra host til `3000` i containeren. 
 `expose` er for eksponering mellem services kun og kan ikke ses af hosten eller omverdenen. Ved ikke om det kan fikse vores problem med loginnet der kun vil kontakte lokalt, men har prøvet at eksponere 3000 og 3002 mellem de to, så de kan kommunikere sammen over de to.